packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -240,70 +277,25 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
     .mdc-textfield__input {
       padding-bottom: 0;
```

Comment:
Check if this is still necessary after removing the 7px padding; I suspect it isn't.

Reply:
It actually turns out the label had a different `bottom` style based on disabled or not. 🙄 

That's fixed and now the problem goes away. Still going through seeing if these styles still need to exist